source/PlayerInfo.cpp

```diff
@@ -2090,6 +2111,28 @@ set<string> &PlayerInfo::Collapsed(const string &name)
 
 
 
+bool PlayerInfo::GetCached(const System *object, std::map<const System*, bool> &cache, bool (PlayerInfo::*func)(const System*) const) const
+{
+	auto it = cache.find(object);
+	if (it == cache.end())
+	{
+		bool value = (this->*func)(object);
+		cache.insert(make_pair(object, value));
```

Comment:
Why do you use `cache.insert(make_pair(object, value))` instead of `cache.emplace(object, value)`?

Note that the generated code is identical, so it doesn't matter.
I'm just curious to why you choose to be more verbose.

Reply:
Wasn't aware of that C++11 addition. Fixed.